cmd/nfsplugin/main.go

```diff
@@ -55,6 +56,8 @@ func main() {
 	cmd.PersistentFlags().StringVar(&endpoint, "endpoint", "", "CSI endpoint")
 	cmd.MarkPersistentFlagRequired("endpoint")
 
+	cmd.PersistentFlags().Uint32Var(&perm, "mount-permissions", 0750, "mounted folder permissions")
```

Comment:
Does `PersistentFlags` parse octal numbers correctly? If not, use string and parse it afterwards.
In addition, you should ensure that the value covers only chmod-able bits (owner/group/other + suid/sgid that may be useful).

Reply:
yeah, now this is a string that we parse later